src/vs/workbench/contrib/editSessions/browser/editSessions.contribution.ts

```diff
@@ -596,7 +596,7 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		'workbench.experimental.editSessions.enabled': {
```

Comment:
should the `experimental` part of the setting key be removed and also the `experimental` tag since it's now on by default?

Reply:
Keeping it on for insiders only just to get additional mileage on the feature set, but may disable it again for 1.71.